calculator.c

```diff
@@ -0,0 +1,80 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+#define MAX 100
+
+// Complete structure of the code is explained in comments
```

Comment:
Let's not write unnecessary comments. Code should be self explanatory and it should not require comments.